rhombus/private/port.rkt

```diff
@@ -21,7 +21,9 @@
    Output
    [current_input current-input-port]
    [current_output current-output-port]
-   [current_error current-error-port]))
+   [current_error current-error-port]
+   [open_output_string open-output-string]
+   [get_output_string get-output-string]))
```

Comment:
These functions should ideally use `define/arity`, especially `get_output_string` because Rhombus wants immutable strings.  But I haven’t had the time to look into filling these in yet.

Reply:
I've done a bit on this—I need some help with the `#:static-infos` key though.